onboarding-enabler-spring/src/main/resources/onboarding-enabler-spring-messages.yml

```diff
@@ -0,0 +1,23 @@
+messages:
+
+    - key: apiml.enabler.registration.successful
+      number: ZWEA001
+      type: INFO
+      text: "Registering to API Mediation Layer: {baseUrl=%s, ipAddress=%s, discoveryServiceUrls=%s}"
+
+    - key: apiml.enabler.registration.initial
+      number: ZWEA002
+      type: INFO
+      text: "Registering to API Mediation Layer: {baseUrl=%s, ipAddress=%s, discoveryServiceUrls=%s}"
+
+    - key: apiml.enabler.registration.renew
+      number: ZWEA003
+      type: INFO
+      text: "Already registered to API Mediation Layer. Will Unregister then register with actual settings: Old values:\
+   {baseUrl=%s, ipAddress=%s, discoveryServiceUrls=%s}; New values: {baseUrl=%s, ipAddress=%s, discoveryServiceUrls=%s}"
+
+    - key: apiml.enabler.registration.fail
+      number: ZWEA004
+      type: ERROR
+      text: "Registering to API Mediation Layer failed: {baseUrl=%s, ipAddress=%s, discoveryServiceUrls=%s}\
+   failed with exception %s"
```

Comment:
If you want to document the codes with the automatic documentation generator, you need to add `reason` and `action` fields. See any other message file